src/exception.jl

```diff
@@ -180,9 +180,9 @@ function pyraise(e::PyError, ::Vector = [])
     ccall((@pysym :PyErr_Restore), Cvoid, (PyPtr, PyPtr, PyPtr),
           e.T, e.val, e.traceback)
     # refs were stolen
-    setfield!(e.T, :o, C_NULL)
-    setfield!(e.val, :o, C_NULL)
-    setfield!(e.traceback, :o, C_NULL)
+    setfield!(e.T, :o, PyPtr_NULL)
+    setfield!(e.val, :o, PyPtr_NULL)
+    setfield!(e.traceback, :o, PyPtr_NULL)
```

Comment:
They are all `PyObject` so `.o` has to be of type `PyPtr`.

Reply:
This is fine, but I thought `setfield!` automatically called `convert` (JuliaLang/julia#16195)?

Reply:
It looks like that's not the case https://github.com/JuliaLang/julia/issues/16195#issuecomment-480089197